lib/src/widgets/chat/ds_image_message_bubble.widget.dart

```diff
@@ -8,8 +8,8 @@ class DSImageMessageBubble extends StatelessWidget {
   final DSAlign align;
   final String url;
   final List<DSBorderRadius> borderRadius;
-  final String imageTitle;
-  final String? imageText;
+  final String? title;
```

Comment:
<img width="657" alt="image" src="https://user-images.githubusercontent.com/753061/189722273-7899afbe-d9ca-4c58-868b-3b7f71712602.png">

Reply:
You shouldn't remove this requirement, since this infos are required by the image widget. So, I suggest to revert the changes done in this file.

<img width="416" alt="Screen Shot 2022-09-12 at 16 14 44" src="https://user-images.githubusercontent.com/91232521/189737026-0d8af445-da76-4801-8cd8-b6fdd622c17b.png">

Reply:
I'm sorry. My previous comment was wrong as I confused it with the video card, where the PR focus is. In my opinion, the picture card shouldn't even have been changed. This change was not made by me. I will investigate why it was made.

Reply:
Done